.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 executors:
   default:
     docker:
-    - image: cimg/go:1.17.9
+    - image: cimg/go:1.18.1
```

Comment:
go 1.18.2 is out, it is just a matter of time to have this image updated. fyi

Reply:
for a next PR :wink: